src/node.cc

```diff
@@ -3475,6 +3537,12 @@ void LoadEnvironment(Environment* env) {
   // Add a reference to the global object
   Local<Object> global = env->context()->Global();
 
+  Local<Object> js_array_object = global->Get(
+      FIXED_ONE_BYTE_STRING(env->isolate(), "Array")).As<Object>();
+  Local<Function> js_array_from_function = js_array_object->Get(
+      FIXED_ONE_BYTE_STRING(env->isolate(), "from")).As<Function>();
+  env->set_array_from(js_array_from_function);
```

Comment:
Ah yeah, I think this is legacy code from when I used to walk the results array in JS.